### PR TITLE
scheduler single and fixes

### PR DESCRIPTION
**Pull request by MichaelDvP**
I've split in small commits.

### CODE REVIEW

File: src/command.cpp [hunk 1]
```diff
@@ -353,6 +353,8 @@ uint8_t Command::call(const uint8_t device_type, const char * cmd, const char *
         return CommandRet::ERROR;
     }
 
+    output.clear(); // we have a command function, clear messages from device_value_info
```
Comment (proddy) [inline src/command.cpp]: don't think its needed, but does no harm

Comment (MichaelDvP) [reply src/command.cpp]: Oh, this was from my first solution for Thomas, not needed anymore after merging your changes.

Comment (proddy) [reply src/command.cpp]: no problem, we can change later

Comment (MichaelDvP) [reply src/command.cpp]: I see that i've already removed it while merging your changes in https://github.com/emsesp/EMS-ESP32/pull/1899/commits/a8307bddbec4b3bfbc165653fd03d6a8c0c83ef4

**Pull request merged**